tissue/check.php

```diff
@@ -1,5 +1,5 @@
 <?php
-$host="localhost"; // Host name
+$host="plantisc"; // Host name

```

Comment:
Why are you changing it? The database host is the same machine where is the portal.
Leave localhost, please.